roles/openshift_control_plane/files/scripts/docker/master-exec

```diff
@@ -14,13 +14,12 @@ if [[ ! -d /etc/origin/node/pods || -z "$( ls -A /etc/origin/node/pods )" ]]; th
   exec "${@:3}"
```

Comment:
Actually don't we need a separate script for crio and docker? One of the requirements that was discussed was that this should work even when openshift-node is down and dockershim goes down with it so we still need to call docker directly when runtime is docker.

Reply:
The rationale to keep two sets of scripts is that crictl version has not been tested by QE. 

It would be nice to have these merged in 3.10.z though. It would require Atomic hosts to have a copy of `crictl` in PATH, the same way we handle `oc` binary